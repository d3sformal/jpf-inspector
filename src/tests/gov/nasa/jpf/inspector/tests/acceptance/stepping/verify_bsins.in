commence
step_transition && wait
back_step_instruction && wait