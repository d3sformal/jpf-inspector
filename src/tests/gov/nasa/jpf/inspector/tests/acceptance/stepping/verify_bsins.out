INFO: JPF created and connected, SuT is started
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=1 hit_count<=1 instruction_type=any]
	SuT (Thread=0) executes the gov/nasa/jpf/inspector/tests/acceptance/sut/MultiTransitionSinglePath.java:0 - invokestatic gov.nasa.jpf.inspector.tests.acceptance.sut.MultiTransitionSinglePath.main([Ljava/lang/String;)V
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/vm/Verify.java:-1 (executenative JPF_gov_nasa_jpf_vm_Verify.getInt__II__I), source unavailable
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/vm/Verify.java:-1 (executenative JPF_gov_nasa_jpf_vm_Verify.getInt__II__I), source unavailable
INFO: JPF Terminating